app/models/patient_session.rb

```diff
@@ -64,4 +64,11 @@ def able_to_vaccinate?
     !unable_to_vaccinate? && !unable_to_vaccinate_not_assessed? &&
       !unable_to_vaccinate_not_gillick_competent?
   end
+
+  def latest_consents_for_each_parent
```

Comment:
I'm not sure the `_for_each_parent` is adding much:

```suggestion
  def latest_consents
```

Reply:
Ya, I went back and forth on this. Will shorten